AFTests/AlgoStore/AlgoStoreInstanceDataTests.cs

```diff
@@ -28,7 +29,9 @@ public partial class AlgoStoreTests : AlgoStoreTestDataFixture
 
             GetPopulatedInstanceDataDTO getinstanceAlgo = new GetPopulatedInstanceDataDTO();
 
-            InstanceDataDTO instanceForAlgo = getinstanceAlgo.returnInstanceDataDTOInvalidAssetPair(algoID);
+            WalletDTO walet = await CreateTestWallet();
+
+            InstanceDataDTO instanceForAlgo = getinstanceAlgo.returnInstanceDataDTOInvalidAssetPair(algoID, walet);
```

Comment:
Line 32 to 34 can be moved to a method since they are used in several other places as well